README.md

```diff
@@ -17,19 +17,17 @@ Implement the `Session` entity:
 
 import { ISession } from "connect-typeorm";
 import { Column, Entity, Index, PrimaryColumn } from "typeorm";
-import { Bigint } from "typeorm-static";
-
 @Entity()
 export class Session implements ISession {
   @Index()
-  @Column("bigint", { transformer: Bigint })
+  @Column("bigint")
```

Comment:
Bigint is simply `{ from: Number, to: Number }`, because PostgreSQL returns the numeric timestamp as a string, where a JS Number is both enough and needed. Without this ValueTransformer, apps will break if they use a different database than MySQL or SQLite. I agree that inlining Bigint with a comment explaining this, and removing the typeorm-static import, will make the README simpler. Please consider changing your patch this way.

Reply:
Hi @nykula! thank you for your quick answer.  I'm using PostgreSQL and the app works fine without the transformer.  The value `expiredAt` is stored as int8 and retrieved as a number. 
I'm reading the max `number` value in typescript is `9223372036854775807`.
Are you storing the `expiredAt` column with milliseconds or seconds?